docs/devsite-help/troubleshooting.md

```diff
@@ -218,3 +218,42 @@ native libraries are copied appropriately.
 
 This information is now in the [protocol buffers guide](protobuf.md#repeated-fields-and-maps); this section has been
 retained for the sake of existing links.
+
+## How can I get ErrorInfo from RpcException?
+
+`ErrorInfo` in `RpcException` provides a consistent and human readable error details.
+It contains three major fields - reason, domain and metadata described as below:
+
+- `Reason`: the proximate cause of the error. This is a stable unique identifier for this
+   type of error within the domain, and can be used to programmatically determine
+   how to handle the error.
+- `Domain`: the logical grouping to which the error belongs. The error domain
+   is typically the registered service name of the tool or product that
+   generates the error. Example: "pubsub.googleapis.com". If the error 
+   is generated by some common infrastructure, the error domain will be 
+   a globally unique value that identifies the infrastructure. 
+   For Google API infrastructure, the error domain is "googleapis.com".
+- `Metadata`: additional structured details about the error.
+
+```csharp
+using Google.Api.Gax.Grpc;
```

Comment:
This example still has unconventional indentation - it looks like it's three spaces for the first indent, and two for the second.

Reply:
Reformatted the indentation.